doc/source/reference/arrays.interface.rst

```diff
@@ -243,6 +244,11 @@ flag is present.
    returning the :c:type:`PyCapsule`, and configure a destructor to decref this
    reference.
 
+.. note::
+
+    :obj:`__array_struct__` is considered legacy and should not be used for new
+    code. Use the :py:doc:`buffer protocol <c-api/buffer>` instead.
```

Comment:
Can we deprecate it? I think also ndarray.__dlpack__ could be mentioned as an alternative, although I see it is not part of the documentation ...

Checking for uses of `__array_struct__` I found only pygame, and opened pygame/pygame#2949 to ask about removing it

Reply:
Sounds like a good idea to me

Reply:
Should I add a deprecation note here? Or wait until it is actually marked for deprecation?